test/vm.install

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 # image-customize script to enable cockpit in test VMs
 # The application RPM will be installed separately
-set -eu
+set -eux
+
+# Update lorax-composer using the updates-testing repository
+if [[ $(grep -ish "Fedora" /etc/redhat-release) == Fedora* ]] && [[ $(grep -ish "firefox" /tmp/BROWSER == "firefox") ]]; then
```

Comment:
`[[` is a bashism, and this is a /bin/sh script. Let's please not break it on Debian-based systems. :-)

This can be done more elegantly with `if grep -iq fedora /etc/redhat-release && ...`.

Reply:
Thanks @martinpitt, updated!